sr_robot_launch/scripts/sr_ur_release

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/python
+# Copyright 2020 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import rospy
+from std_msgs.msg import Bool
+from ur_dashboard_msgs.srv import GetSafetyMode, GetProgramState, GetRobotMode, Load
+from ur_dashboard_msgs.msg import SafetyMode, ProgramState, RobotMode
+from std_srvs.srv import Trigger
+
+class SrUrUnlock():
+    def __init__(self):
+        self.external_control_program_name = "external_ctrl.urp"
+        self.arms = []
+        if rospy.has_param('ra_sr_ur_robot_hw'):
+            self.arms.append('ra')
+        if rospy.has_param('la_sr_ur_robot_hw'):
+            self.arms.append('la')
+        if len(self.arms) == 0:
+            rospy.signal_shutdown("No arms detected, shutting down %s", rospy.get_name())
+        self.subscriber = rospy.Subscriber("sr_arm/release", Bool, self.pedal_sub)
+        self.rate = rospy.Rate(5)
+        while not rospy.is_shutdown():
+            try:
+                self.rate.sleep()
+            except rospy.ROSInterruptException:
+                rospy.loginfo("Shutting down %s", rospy.get_name())
+
+    def pedal_sub(self, data):
+        if data == True:
+            self.release_arm()
+
+    def call_dashboard_service(self, side, service_name):
+        service_string = "/" + side + "_sr_ur_robot_hw/dashboard/" + service_name
+        serv_call = rospy.ServiceProxy(service_string, Trigger)
+        resp = serv_call()
+
+    def call_service(self, side, service_name):
+        service_string = "/" + side + "_sr_ur_robot_hw/" + service_name
+        serv_call = rospy.ServiceProxy(service_string, Trigger)
+        resp = serv_call()
+
+    def get_safety_mode(self, arm):
```

Comment:
in all these you can reuse call_dashboard_service, just make it return resp value and ignore it if not needed

Reply:
df3a852d7bf5e5a16fda42d3fffb5f85ab6e8adc